index.html

```diff
@@ -11,7 +11,7 @@
   <body>
    
     <header>
-      <a href="./index.html"><img class="logo" src="./public/logo.png" alt=""> </a>
+      <a href="./index.html"><img class="logo" src="./public/logo.png" alt="logo"> </a>
```

Comment:
Please wrap your `img` tag in a `<figure> ` tag for better accessibility

Reply:
Okay Thank you